index.html

```diff
@@ -0,0 +1,571 @@
+- the names of any fields the spec introduces on *foreign types* defined in a different feature
+- the names of any arguments the spec introduces on *foreign directives and fields* defined by a different feature
+- the names of any directives introduced in the spec, with the exception of the *root directive*, which must have the same name as the feature
+
+<view-of src=prefixing.graphql
+  title='Prefixing examples'
+></view-of>
+
+## Versioning
+
+Specs are versioned with a **subset** of a [Semantic Version Number](https://semver.org/spec/v2.0.0.html) containing only the `major` and `minor` parts. Thus, specifications **should** provide a version of the form `‹major›.‹minor›`, where both integers >= 0.
+
+```
+#::[Examples of valid versions]
+2.2
+1.0
+1.1
+0.1
+```
+
+As specified by semver, spec authors **should** increment the:
+> - MAJOR version when you make incompatible API changes,
+> - MINOR version when you add functionality in a backwards compatible manner
+
+Patch and pre-release qualifiers are judged to be not particularly meaningful in the context of core features, which are (by definition) interfaces rather than implementations. The `‹patch›` component of version denotes a bug fix which is backwards compatible—that is, a change to the implementation which does not affect the interface. `‹patch›`-level changes in the version of a spec denote wording clarifications which do not require implementation changes. As such, it is not important to track them for the purposes of version resolution.
+
+As with [semver](https://semver.org/spec/v2.0.0.html), the `0.x` version series is special: there is no expectation of compatibility between versions `0.x` and `0.y`. For example, a processor must not activate implementation `0.4` to satisfy a requested version of `0.2`.
+
+### Ordering
+
+Versions can be ordered with the following pseudocode:
+
+```typescript
+compareVersions(a: Version, b: Version) -> LESS_THAN | EQUAL | GREATER_THAN {
+  if a.major < b.major {
+    return LESS_THAN
+  }
+  if a.major == b.major {
+    if a.minor == b.minor { return EQUAL }
+    if a.minor > b.minor { return GREATER_THAN }
+    return LESS_THAN
+  }
+  if a.major > b.major {
+    return GREATER_THAN
+  }
+}
+```
+
+Note that this ordering does *not* guarantee that if A > B then A can satisfy a version requirement of B. For example, `2.0 > 1.9`, but `2.0` cannot [satisfy](#satisfying-version-requirements) the requirements of a document which references version `1.9`, since the major version update introduced possibly-breaking changes.
+
+### Satisfaction
+
+Given a version `requested` by a document and an `available` version of an implementation, the following pseudocode will determine if the `available` version can satisfy the `requested` version:
+
+```typescript
+canSatisfy(requested: Version, available: Version) =
+  requested.major == available.major &&
+  (requested.major > 0 && (requested.minor <= available.minor)
```

Comment:
I think an extra pair of parentheses around the two arguments to this `&&` would be nice, even if it's not required by standard precedence rules